.gitignore

```diff
@@ -122,3 +122,7 @@ github_utils.py
 
 #Settings secrets
 secret.py
+
+#Autogenerated files
+src/app/media/
+src/spdx-license-matcher/
```

Comment:
For me, spdx-license-matcher got downloaded and installed in the virtualenv. Are you getting `src/spdx-license-matcher/` folder? When is this folder created?

Reply:
Yes. Mainly this folder is mentioned in `requirements.txt` file so when we do `pip install -r requirements.txt`, it got installed.

Reply:
I guess I forgot to use virtualenv during pip install. Sorry about that. I had removed that file from gitignore as it is installed in `venv` folder which is already present in `.gitingore`